### DISCUSSION TITLE
[A-L] (26 Gennaio 2024) Esercizio 4

### DISCUSSION THREAD

**Discussion by FeddyLix17** [sapienzastudentsnetwork/metodi-matematici-per-l-informatica#402] [Esami A-L]
Trovare l’errore (o gli errori) nella seguente dimostrazione per induzione forte. [5 punti]

- Tesi: Per ogni $\Large n \geq 0, 7 \times n = 0$.

- Base: Se $\Large n = 0$ allora $\Large 7 \times n = 7 \times 0 = 0$.

- Passo: Assumiamo che la tesi sia vera per tutti i numeri da 0 a $\Large n$, per un generico intero $\Large n \geq 0$. Dimostriamo che è vera per $\Large n + 1$. 

    Possiamo scrivere $\Large n + 1 = a + b$ con $\Large a,\ b$ interi $\Large 0 \leq a, b \leq n$.
    
    Per Ipotesi Induttiva abbiamo $\Large 7 \times a = 0\ \texttt{e}\ 7 \times b = 0$.
    
    Dunque $\Large 7 \times (n + 1) = 7 \times (a + b) = 7 \times a + 7 \times b = 0 + 0 = 0$.

NB: non è sufficiente dire che la tesi dimostrata è falsa - questo è ovvio. Indicare esplicitamente se il Caso Base e il Passo Induttivo sono corretti.

> l'esercizio è uguale per entrambe le file

**Comment by DavideGalilei** [comment]
Soluzione presente qui: https://sasholo.notion.site/ce9d37680c27447bbe91632ec3d9938b?v=3c5601b02590483b8f915e253779e275&p=d10f2fda01954c25a165b546918dd702&pm=s

Caso base OK, Passo induttivo no.
Non posso scrivere $n+1 = a+b$ perchè se dovessi avere $n= 0$, 
$0+1=a+b \implies \nexists a,b \ | 0\le a,b \le n$, cosa che invece è sostenuta dalla traccia.

Questo perché se $0\le a,b \le n$ con $n=0$, non esiste una coppia $a, b$ tale che $0\le a,b \le 0$ e che $a + b = 1$
